ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -241,12 +276,6 @@ function check_clusteroperators() {
     done < "${input}"
     rm -f "${tmp_clusteroperator}"
 
-    echo "Make sure every operator column reports version"
```

Comment:
Any background to remove this check?

Reply:
This step is covered in `Make sure every operator's AVAILABLE column is True`.

Reply:
I think the step "Make sure every operator's AVAILABLE column is True"  is checking "AVAILABLE" column, while this step is checking "VERSION" column.

Reply:
my bad, a wrong paste, i meant it's covered in `"Make sure every operator do not report empty column"`

Reply:
Got it. Thanks.

Reply:
Hmm, assuming `VERSION` field is empty, but we check `AVAILABLE` by using `if unavailable_operator=$(${OC} get clusteroperator | awk '$3 == "False"' | grep "False")`, then we will consider `PROGRESSING` status as `AVAILABLE` status by mistake.

```
# oc get co
NAME                                       VERSION                              AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
authentication                              False        True         False      14m     
baremetal                                      False        True         False      33m     
```

Reply:
@shellyyang1989 you are talking about the step `Make sure every operator's AVAILABLE column is True`, right? I've corrected myself in https://github.com/openshift/release/pull/43227/files#r1339475755. It should be `Make sure every operator do not report empty column`